### PR TITLE
[TO DELETE] Make Sinogram Docs Numpy-Style and Move to Labs [new]

### PR DESCRIPTION
**Pull request by kingrpaul**
Undo edits

### CODE REVIEW

**Comment by kingrpaul** [comment]
@SimonBiggs Formatted the sinogram docstrings and moved this into the labs. Would appreciate a review and merge. Note that there is a just-previous commit that swept up some things I didn't intend and cannot account for. So, i back-ed up one commit and made this one. Kindly disregard the other, I'll delete it after the merge. Thanks.

File: src/pymedphys/sinogram/_level1/sinogram.py [hunk 1]
```diff
@@ -39,22 +39,34 @@
```
Comment (SimonBiggs) [inline src/pymedphys/sinogram/_level1/sinogram.py]: This file exists both in the labs and here? Where would you like it to be?

Comment (kingrpaul) [reply src/pymedphys/sinogram/_level1/sinogram.py]: @SimonBiggs Thought I had deleted the duplicate. The goal is to move it into the labs. Thanks.

**Comment by SimonBiggs** [comment]
Kk, once I'm at the computer I can delete it out of the repo and merge this in.

**Comment by SimonBiggs** [comment]
So although sinogram was moved to labs there was a little bit of a trail that needs updating. The labs docs needs to point to the labs version, and the sinogram tests also need to point to the labs versions.

**Comment by kingrpaul** [comment]
> Need to point docs and tests to the new sinogram location
Thank you @SimonBiggs. Got it. Waiting for the pipeline.

**Comment by SimonBiggs** [comment]
Tests are fixed, docs are still importing it somewhere...

**Comment by SimonBiggs** [comment]
Also, when you go `git add -A` always run `git status` afterwards. You're going to have to repoint this branch to its previous commit.

I believe when you're running the docs build command you're doing it to the wrong directory...

**Comment by SimonBiggs** [comment]
> I believe when you're running the docs build command you're doing it to the wrong directory...

Potentially you're not using the correct orientation of slashes for the OS you are on ...

**Comment by SimonBiggs** [comment]
Actually, scrap that, even on windows you run the code with forward slashes:

```
sphinx-autobuild -W -p 7070 docs docs/_build/html
```

**Comment by SimonBiggs** [comment]
You can't just delete the files and move on. If you do they are still permanently added to the git history, taking up unnecessary space. You'll need to do:

```
git checkout e22a9e59f167278e83810fa9dda4a662049fd177
git checkout -b new-branch-name
git push
```

**Comment by SimonBiggs** [comment]
By the way, here are the files I am referring to:

https://github.com/pymedphys/pymedphys/pull/150/files

**Comment by SimonBiggs** [comment]
@kingrpaul before continuing too much further you do need to resubmit this pull request from a commit that doesn't inherit from that which introduced all the excess files.

**Comment by kingrpaul** [comment]
> sphinx-autobuild -W -p 7070 docs docs/_build/html

from www.pymedphys.com?
     `sphinx-autobuild docs docs/_build/html`

**Comment by SimonBiggs** [comment]
Closed in favour of https://github.com/pymedphys/pymedphys/pull/157

**Pull request closed without merging**